docs/source/configuration/external_timelock_service_configs/timelock_server_config.rst

```diff
@@ -0,0 +1,136 @@
+.. _timelock-server-configuration:
+
+Timelock Server Configuration
+=============================
+
+The Timelock Server configuration file is written in YAML and is located at ``var/conf/timelock.yml``.
+It has three main configuration blocks: ``clients``, ``cluster``, and ``algorithm``. We will discuss how each of
+these may be configured in turn, as well as additional configuration parameters.
+
+.. contents::
+   :local:
+
+Clients
+-------
+
+The ``clients`` block is a list of strings which corresponds to client namespaces that the server will respond to.
+At least one client must be configured for the Timelock Server to work. Querying an endpoint for a client that does not
+exist will result in a 404.
+
+   .. code:: yaml
+
+      clients:
+        - tom
+        - jerry
+
+A single Timelock Server or cluster of Timelock Servers can support multiple AtlasDB clients. When querying a
+Timelock Server, clients must supply a namespace for which they are requesting timestamps or locks in the form of a
+path variable. There are no guarantees of relationships between timestamps requested from different namespaces.
+
+   .. code:: bash
+
+      curl -XPOST localhost:8080/tom/timestamp/fresh-timestamp
+      curl -XPOST localhost:8080/jerry/timestamp/fresh-timestamp # no guarantees of any relationship between the values
+
+This is done for performance reasons: consider that if we maintained a global timestamp across all clients, then
+requests from each of these clients would need to all be synchronized.
+
+Cluster
+-------
+
+.. note::
+
+   You will probably want to use an odd number of servers; using an even number of servers increases the overhead
+   of distributed consensus while not actually providing any additional fault tolerance as far as obtaining a quorum
+   is concerned. Using more servers leads to improved fault tolerance at the expense of additional overhead incurred
+   in leader election and consensus.
+
+The ``cluster`` block is used to identify the servers which make up a Timelock Service cluster. An example is as
+follows:
+
+   .. code:: yaml
+
+      cluster:
+        localServer: palantir.com:8700
+        servers:
+          - palantir.com:8700
+          - palantir.com:8701
+          - palantir.com:8702
```

Comment:
I've changed to your suggestion - yes, as you've mentioned there might be the risk that someone reads it as collocating them!

The default port we've settled on isn't 8700; Dropwizard's default is 8080 so I've changed this example to that, though internally we might use something different to avoid clashes.